intrinsicMatrixL: 
 first: [913.5889,0.7017,671.1454]
 second: [0,913.7067,517.8589]
 third: [0,0,1]
distortionMatrixL: [0.0124,0.0575,0.0029,-0.0011,-0.1556]
intrinsicMatrixR: 
 first: [907.2019,1.5681,669.2461]
 second: [0,907.8925,501.1368]
 third: [0,0,1]
distortionMatrixR: [0.0348,-0.0798,0.0027,-0.00019,0.0467]
transMatrix:
 first: [0.9999,-0.0100,-0.0085,-65.1455]
 second: [0.0101,0.9999,0.0098,-0.2843]
 third: [0.0084,-0.0099,0.9999,-0.5559]
